tutorials/rendering/jitter_stutter.rst

```diff
@@ -147,6 +147,224 @@ iOS
+
+**Solution (Windows):** Set the Power Profile to "High Performance":
+
+.. image:: img/windows-high-performance-profile.webp
+
+.. tip::
+
+    Sometimes "CPU Boost" where a single CPU core goes into overdrive can be a problem.
+    You can prevent CPU Boost by editing your *current* energy plan and set
+    **Processor power management > Maximum processor state** to 99% instead of 100%:
+
+    .. image:: img/windows-no-boost.webp
+
+For NVIDIA on Windows, set "Power management mode" on "Prefer maximum performance".
+
+.. image:: img/nv-power-profile.webp
+
+**Solution (Linux):** Set the CPU governor to Performance:
+
+.. code:: sh
+
+    sudo cpupower frequency-set -g performance
+
+    # AMD
+    echo high | sudo tee /sys/class/drm/card1/device/power_dpm_force_performance_level
+
+You can try `CoreCtrl <https://gitlab.com/corectrl/corectrl>`__ or `Feral GameMode <https://github.com/FeralInteractive/gamemode>`__
+for user-friendly GUI to control power profiles for individual applications.
+
+On NVIDIA, open **nvidia-settings** and go to **GPU 0 > PowerMizer > Preferred Mode: Prefer Maximum Performance**.
+
+.. image:: img/nv-linux-powermizer.webp
+
+Windows: Unlicensed
+~~~~~~~~~~~~~~~~~~~
+
+First of all, "unactivated" or "unlicensed" Windows can have arbitrary limitations imposed by Microsoft.
+These limitations can change over time and an important one is that Microsoft may decide to restrict
+applications from entering "Hardware Independent Flip" modes which provides the lowest latency experience
+while in fullscreen.
+
+Second, even if Microsoft decides to not artificially limit "Hardware Independent Flip"; if
+you're seeing this watermark:
+
+.. image:: img/activate-windows.webp
+
+Chances are you're not running in "Hardware Independent Flip" because Windows must use DWM
+compositing to display the watermark on top of Godot. And DWM Compositing prevents entering the desired mode.
+
+**Solution:** Activate Windows by buying an original license key.
+
+Windows: Presentation Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Windows has many ways of presenting to the screen. `Special K has an in-depth explanation of them <https://wiki.special-k.info/Presentation_Model>`__
+
+But we can basically reduce them to the following:
+
+**Good**
+
+1. Hardware: Independent Flip (also called "iFlip").
+2. Hardware: Legacy Flip.
+3. Hardware Composed: Independent Flip.
+
+**Bad**
+
+1. Composed: Flip.
+
+You can use `PresentMon <https://github.com/GameTechDev/PresentMon/releases>`__ to see which mode Godot is in.
+While in Fullscreen, Godot should be in one of the "good" ones.
+
+If Godot is not in one of the "bad" ones, there may be an overlay that is causing problems
+(e.g. AMD Overlay, NVIDIA's overlay, Microsoft Game Bar, "Activate Windows" watermark, etc).
+
+Godot can only enter the good ones if it's fully covering a single monitor with nothing on top.
+Some newer HW may be able to enter "Hardware Composed: Independent Flip" even if not fullscreen though.
```

Comment:
```suggestion
Godot can only enter the good ones if it's fully covering a single monitor with nothing on top.
Some newer hardware may be able to enter "Hardware Composed: Independent Flip" even if not fullscreen though, thanks to support for multiple plane overlays (MPO).
This behavior can sometimes be unreliable (e.g. it may vary depending
on monitor count or whether HDR is enabled), so it's still better to
use fullscreen whenever possible.
```